project/LinkerdBuild.scala

```diff
@@ -12,7 +12,8 @@ object LinkerdBuild extends Base {
   val Bundle = config("bundle") extend Minimal
   val Dcos = config("dcos") extend Bundle
 
-  val consul = projectDir("consul")
+  lazy val consul = projectDir("consul")
```

Comment:
We try to avoid lazy vals in our build file to keep dependancies explicit.  Adding a dependency on namer/core actually tangles things up a bit.  I actually think the best way to resolve this is to move the CatalogNamer out of `consul` and into `namer/consul`.